server/tests/integration-tests/metadata.test.js

```diff
@@ -0,0 +1,179 @@
+import request from 'supertest-as-promised'
+import httpStatus from 'http-status'
+import chai, { expect } from 'chai'
+import app from '../../..'
+import mongoUnit from 'mongo-unit'
+
+chai.config.includeStack = true
+
+describe('## Metadata APIs', () => {
+
+  const testMongoUrl = process.env.MONGO_HOST
+  const testData = require('./fixtures/testData.json')
+
+  before(() => mongoUnit.initDb(testMongoUrl, testData))
+  after(() => mongoUnit.drop())
+
+  const validUserCredentials = {
+    email: 'test@test.de',
+    password: 'asdf'
+  }
+
+  let metadata = {
+      _id: "religion",
+      data: "{\"south_arabian\":[\"South Arabian\",\"rgb(153,26,51)\",\"Arabian_mythology\",\"Paganism\"],\"ashurism\":[\"Ashurism\",\"rgb(230,230,230)\",\"Ancient_Mesopotamian_religion\",\"Paganism\"],\"african\":[\"African\",\"rgb(204,5",
+      __v: 0,
+      score: 0,
+      type: "g",
+      coo: []
+
+  }
+
+  let updateMarker = {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this useless assignment to local variable "updateMarker" ([javascript:S1854](https://sonarcloud.io/organizations/aumanjoa-github/rules#rule_key=javascript%3AS1854))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=chronas-api&pullRequest=21&issues=AWWk8ELXfVFgtsXauFlQ&open=AWWk8ELXfVFgtsXauFlQ)

[ISSUE_KEY]: AWWk8ELXfVFgtsXauFlQ